src/pages/Homepage/index.js

```diff
@@ -63,7 +64,8 @@ const Homepage = () => {
           <div className="landing-actions">
             <h1>Wonder Bakery</h1>
             <p className="text-md">Fresh baked goods made with love</p>
-            <button className="btn btn-primary">Order Now</button>
+            {/* <button className="btn btn-primary">Order Now</button> */}
```

Comment:
I think you forgot to remove this comment. If it is not necessary, do remove this.